lib/themes/base/tokens.ts

```diff
@@ -83,10 +83,14 @@ const makeColourGamut = () =>
 
 export const tokens: Tokens = {
 	breakpoints: {
-		mobile: 650,
-		tablet: 651,
-		desktop: 1001,
-		largeDesktop: 1201,
+		mobile: 0, // TODO: Check. Can this just be 0?
+		tablet: 768,
```

Comment:
Checked the 650 and it was inline with previous breakpoint reviews but I think it can be increased to 690 at least but 768 captures new iPad but misses some smaller tablet devices that may or may not be a a problem
<img width="448" alt="Tablets" src="https://user-images.githubusercontent.com/1334204/77129393-9d529200-6a9f-11ea-926e-9bc06c130d6b.png">